examples/async/parent_child.py

```diff
@@ -161,8 +159,6 @@ async def add_answer(
         answer = Answer(
             # required make sure the answer is stored in the same shard
             _routing=self.meta.id,
-            # since we don't have explicit index, ensure same index as self
-            _index=self.meta.index,
```

Comment:
This wasn't necessary on this example, since all the documents belong to the same index. I removed it because it creates a typing problem, because `_index` is stored as an `Index` or `AsyncIndex`, but here it was given as a string.